tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs

```diff
@@ -28,9 +28,9 @@ public NodeDynamicInstallationTest(ITestOutputHelper output) : base(output)
             {
                 var data = new TheoryData<string, string>();
                 var imageTestHelper = new ImageTestHelper();
-                data.Add("12.22.11", imageTestHelper.GetGitHubActionsBuildImage());
-                data.Add("14.19.1", imageTestHelper.GetGitHubActionsBuildImage());
-                data.Add("16.14.2", imageTestHelper.GetGitHubActionsBuildImage());
+                data.Add("12.22.12", imageTestHelper.GetGitHubActionsBuildImage());
```

Comment:
We could add a new constant `final-stretch-node12-version` [here](https://github.com/microsoft/Oryx/blob/main/build/constants.yaml#L114) and run the ./build/generateConstants.sh script. That way, we would be able to use the constant for node 12 instead of hardcoding to be consistent with what we are doing for node 14 and node 16.

Reply:
yah, I considered that. Actually, the versions of node 14,16 for stretch was partially hard coded, I replaced them with the constants. For version I did not add the additional constant as it was not there and for stretch, we will not consider the new versions further.